packages/dd-trace/test/appsec/sdk/track_event.spec.js

```diff
@@ -331,9 +335,509 @@ describe('track_event - Internal API', () => {
         'appsec.events.custom_event.track': 'true',
         '_dd.appsec.events.custom_event.sdk': 'true'
       })
+      expect(waf.run).to.not.have.been.called
       expect(prioritySampler.setPriority)
         .to.have.been.calledOnceWithExactly(rootSpan, USER_KEEP, ASM)
-      expect(telemetry.incrementSdkEventMetric).to.have.been.calledWith('custom')
+      expect(telemetry.incrementSdkEventMetric).to.have.been.calledWithExactly('custom', 'v1')
+    })
+
+    it('should call to the waf when event name is "users.login.success"', () => {
+      trackCustomEvent(tracer, 'users.login.success')
+
+      expect(waf.run).to.have.been.calledOnceWithExactly({
+        persistent: {
+          [LOGIN_SUCCESS]: null
+        }
+      })
+    })
+
+    it('should call to the waf when event name is "users.login.failure"', () => {
+      trackCustomEvent(tracer, 'users.login.failure')
+
+      expect(waf.run).to.have.been.calledOnceWithExactly({
+        persistent: {
+          [LOGIN_FAILURE]: null
+        }
+      })
+    })
+  })
+
+  describe('eventTrackingV2', () => {
+    describe('trackUserLoginSuccessV2', () => {
+      it('should log warning when root span is not available', () => {
+        rootSpan = undefined
+
+        trackUserLoginSuccessV2(tracer, 'login')
+
+        expect(log.warn)
+          .to.have.been.calledOnceWithExactly('[ASM] Root span not available in v2.trackUserLoginSuccess')
+        expect(setUserTags).to.not.have.been.called
+      })
+
+      it('should log warning when passed invalid login', () => {
+        trackUserLoginSuccessV2(tracer, null)
+        trackUserLoginSuccessV2(tracer, {})
+
+        expect(log.warn).to.have.been.calledTwice
+        expect(log.warn.firstCall)
+          .to.have.been.calledWithExactly('[ASM] Invalid login provided to v2.trackUserLoginSuccess')
+        expect(log.warn.secondCall)
+          .to.have.been.calledWithExactly('[ASM] Invalid login provided to v2.trackUserLoginSuccess')
+        expect(setUserTags).to.not.have.been.called
+        expect(rootSpan.addTags).to.not.have.been.called
+        expect(waf.run).to.not.have.been.called
+      })
+
+      it('should call to addTags and waf only with login', () => {
+        trackUserLoginSuccessV2(tracer, 'login')
+
+        expect(log.warn).to.not.have.been.called
+        expect(setUserTags).to.not.have.been.called
+
+        expect(rootSpan.addTags).to.have.been.calledOnceWithExactly(
+          {
+            'appsec.events.users.login.success.track': 'true',
+            '_dd.appsec.events.users.login.success.sdk': 'true',
+            'appsec.events.users.login.success.usr.login': 'login'
+          })
```

Comment:
i know this is just a styling nit, but why not just use the normal formatting that's used everywhere else ?
```suggestion
        expect(rootSpan.addTags).to.have.been.calledOnceWithExactly({
          'appsec.events.users.login.success.track': 'true',
          '_dd.appsec.events.users.login.success.sdk': 'true',
          'appsec.events.users.login.success.usr.login': 'login'
        })
```

Reply:
I'm pretty sure you have the wrong indentation now 
![image](https://github.com/user-attachments/assets/42978291-7124-4c5e-b46d-8f940b049d7e)

Reply:
linter is failing, so I guess so